x-pack/legacy/plugins/ml/public/data_frame/components/group_by_list/popover_form.tsx

```diff
@@ -156,6 +174,17 @@ export const PopoverForm: React.SFC<Props> = ({
       <EuiFormRow
         error={!validAggName && [aggNameError]}
         isInvalid={!validAggName}
+        helpText={
+          isUnsupportedAgg
+            ? i18n.translate(
+                'xpack.ml.dataframe.groupBy.popoverForm.unsupportedAggregationHelpText',
+                {
+                  defaultMessage:
+                    'Besides the custom aggregation name, editing this aggregation is not supported using this form. Please use the advanced editor instead.',
```

Comment:
Is this label referring to the aggregation part of the config, or the group by? If group by, is the text correct? As above, maybe a slight edit to the text to something like `Only the aggregation name can be edited in this form. Please use the advanced editor to edit the other parts of the aggregation.` (and refer to `group_by` and not `aggregation` if necessary).

Reply:
Fixed in a98f94a.